### PR TITLE
Update autocomplete-component.md

### PR DESCRIPTION
**Pull request by twify93**
Fix simple type and missing attribute in the code for index.js autocomplete

### CODE REVIEW

File: source/localizable/tutorial/autocomplete-component.md [hunk 1]
```diff
@@ -102,6 +102,7 @@ Now, define your new controller like so:
 ```app/controllers/index.js
 export default Ember.Controller.extend({
   filteredList: null,
+  model: null,
```
Comment (locks) [inline source/localizable/tutorial/autocomplete-component.md]: Can you amend the commit to remove this line? We generally don't declare the model property explicitly for routable controllers.

**Comment by locks** [comment]
Thanks for the contribution! I left a small note inline, I'll merge once you address it :)

File: source/localizable/tutorial/autocomplete-component.md (hunk 1 above)
Comment (twify93) [inline source/localizable/tutorial/autocomplete-component.md]: But running the script without this line, makes the code throwing error when the code reaches this line:
this.set('model').clear(); on line 21
The error thrown:
> index.js:25 Uncaught TypeError: Cannot read property 'clear' of undefined

The error occures when you search for empty string.

Clone my repo and test it:
[https://github.com/twify93/emberjs-rentals](url)

Comment (locks) [inline source/localizable/tutorial/autocomplete-component.md]: The error is related with [another problem](https://github.com/emberjs/guides/pull/1264), we're currently reviewing the whole guide.

Comment (twify93) [inline source/localizable/tutorial/autocomplete-component.md]: Ok, so I am removing this line and will fix it with .set :)

**Comment by acorncom** [comment]
@homu r+  Thanks!

**Comment by toddjordan** [comment]
@homu r+

**Comment by toddjordan** [comment]
@homu retry

**Comment by toddjordan** [comment]
@twify93 the tests on this timed out for some reason causing the build to fail.  Since then there have been other merges, so if you could rebase with the latest we can try the merge again.

**Comment by twify93** [comment]
OK. I will review the documentation and make additional changes where they are needed.

**Pull request closed without merging**